File: JavaScript Skeleton/models/Film.js
const mongoose = require('mongoose');

let filmSchema = mongoose.Schema({
    name: {type: 'string', require: 'true'},
    genre: {type: 'string', require: 'true'},
    director: {type: 'string', require: 'true'},
    year: {type: Number, require: 'true'}
});

let Film = mongoose.model('Film', filmSchema);

module.exports = Film;

File: C# Skeleton/IMDB/Models/Film.cs
using System.ComponentModel.DataAnnotations;
using System.Web.Mvc;

namespace IMDB.Models
{
    public class Film
    {
        [Key]
        public int Id { get; set; }

        [Required]
        [AllowHtml]
        public string Name { get; set; }

        [AllowHtml]
        [Required]
        public string Genre { get; set; }

        [AllowHtml]
        [Required]
        public string Director { get; set; }

        [Required]
        public int Year { get; set; }
    }
}